doc/modules/changes.h

```diff
@@ -6,6 +6,11 @@
  *
  *
  * <ol>
+ * <li> New: The stokes residual postprocessor will output the convergence
```

Comment:
Maybe put the name of the postprocessor into quotes, to make clear how it is actually called?